examples/experimental/dagster-airlift/dagster_airlift/core/load_defs.py

```diff
@@ -36,7 +37,7 @@
 @dataclass
 class AirflowInstanceDefsLoader(StateBackedDefinitionsLoader[SerializedAirflowDefinitionsData]):
     airflow_instance: AirflowInstance
-    explicit_defs: Definitions
+    mapped_assets: Iterable[MappedAsset]
```

Comment:
could we use `Sequence` instead of `Iterable` across this PR? doesn't seem like we have a strong reason to make it `Iterable` & `Sequence` is more consistent across other parts of codebase

Reply:
yea I buy this.